### PR TITLE
PAYARA-2869 StackTrace Split into Multiple Log Records PAYARA4

### PR DESCRIPTION
**Pull request by MattGill98**

### CODE REVIEW

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by MattGill98** [comment]
jenkins quick test payara 4 please

**Comment by MattGill98** [comment]
jenkins quick test payara 4 please

File: nucleus/common/common-util/pom.xml [hunk 1]
```diff
@@ -75,8 +75,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.7</source>
```
Comment (Cousjava) [inline nucleus/common/common-util/pom.xml]: This could be removed entirely and so inherit from the parent pom.

**Comment by MattGill98** [comment]
jenkins quick test payara 4 please

**Pull request merged**